### PR TITLE
Add Alt-backspace config into example/emacs.yml

### PR DESCRIPTION
**Pull request by zw963**
Add emacs mapping for `Kill word backward` use `Alt-backspace`, and exclude this config for emacs itself too.

Thanks

### CODE REVIEW

File: example/emacs.yml [hunk 1]
```diff
@@ -32,6 +34,8 @@ keymap:
       M-d: [C-delete, { set_mark: false }]
       # Kill line
       C-k: [Shift-end, C-x, { set_mark: false }]
+      # Kill word backward
+      Alt-backspace: [C-backspace, C-x, {set_mark: false}]
```
Comment (k0kubun) [inline example/emacs.yml:38]: In my environment (Ubuntu 22.04, X11 GNOME), `C-backspace` already deletes a word, so `C-x` seems to do nothing. Why did you add `C-x` here?

Comment (zw963) [reply example/emacs.yml]: Sorry, my fault, i modified config from the copy of above `C-k:`  :smile:

I added a C-M-Space config too, please check.

**Comment by zw963** [comment]
Hi, i do propose change `C-s: f3` into `C-s: C-f`, because after user search a keyword use F3 at the first time, then pressing F3(C-s) again, which will search next previous keyword instead of popup the search menu, which is not we want.

Thanks

**Review by k0kubun — APPROVED** [review]
